test/common.test.js

```diff
@@ -0,0 +1,134 @@
+import test from "ava";
+import * as impl from "../js/common.js";
+
+test("inputParser", t => {
+	// Toot ID
+
+	t.notThrows(() =>
+		impl.inputParser(
+			"1",
+			() => t.fail(),
+			tootId => t.is(tootId, "1")
+		)
+	);
+	t.notThrows(() =>
+		impl.inputParser(
+			"103492879513261939",
+			() => t.fail(),
+			tootId => t.is(tootId, "103492879513261939")
+		)
+	);
+	t.notThrows(() =>
+		impl.inputParser(
+			"65tk7_4dby7w3",
+			() => t.fail(),
+			tootId => t.is(tootId, "103492879513261939")
+		)
+	);
+
+	// Toot URL
+
```

Comment:
Toot URL は `/web/statuses/0123` 形式だけじゃなく `/@userid/0123` 形式も許容するように、以下のテストを足してください
```suggestion
	t.notThrows(() =>
		impl.inputParser(
			"https://qiitadon.com/@blhsrwznrghfzpr/103497813217417982",
			() => t.fail(),
			tootId => t.is(tootId, "103497813217417982")
		)
	);
```

Reply:
あっラグってマジがってレビュー要求してしまった、今対応します

Reply:
対応しました